tensorboard/plugins/interactive_inference/utils/inference_utils_test.py

```diff
@@ -456,6 +456,36 @@ def test_convert_predict_response_classification(self):
     self.assertAlmostEqual(
         .2, converted.result.classifications[2].classes[1].score)
 
+  def test_vizparams_pattern_parser(self):
+    viz_params = inference_utils.VizParams(
+        x_min=1,
+        x_max=10,
+        examples=[],
+        num_mutants=0,
+        feature_index_pattern=None)
+    self.assertEqual([], viz_params.feature_indices)
+    viz_params = inference_utils.VizParams(
+        x_min=1,
+        x_max=10,
+        examples=[],
+        num_mutants=0,
+        feature_index_pattern='1-3')
+    self.assertEqual([1, 2, 3], viz_params.feature_indices)
+    viz_params = inference_utils.VizParams(
+        x_min=1,
+        x_max=10,
+        examples=[],
+        num_mutants=0,
+        feature_index_pattern='3, 1')
+    self.assertEqual([1, 3], viz_params.feature_indices)
+    viz_params = inference_utils.VizParams(
+        x_min=1,
+        x_max=10,
+        examples=[],
+        num_mutants=0,
+        feature_index_pattern='0-')
```

Comment:
can we add a bit more test around corner case? 
i.e.,
- `feature_index_pattern='0-1-2'`
- `feature_index_pattern='3-1'`
- `feature_index_pattern='1-Foo'`

Reply:
added more tests